src/shared/java/org/truffleruby/shared/TruffleRuby.java

```diff
@@ -18,27 +18,22 @@
     public static final String MIME_TYPE = "application/x-ruby";
     public static final String EXTENSION = ".rb";
     public static final String ENGINE_ID = "truffleruby";
-    public static final String LANGUAGE_VERSION = "2.7.2";
+    public static final String LANGUAGE_VERSION = "2.7.2"; // BuildInformationImpl.INSTANCE.getLanguageVersion();
```

Comment:
When I switch to the commented code, something in annotation processing breaks. Can anyone help me figure out why?

Reply:
I think this is not worth it (so good as it is before), because it's so much simpler and easier to understand to have the literal version here.
Could you undo that change as well as removing `findLanguageVersion()`?

https://github.com/oracle/truffleruby/blob/master/doc/contributor/updating-ruby.md#make-other-changes says to update both, so there seems to be no risk for them to not be in sync.
Also, `.ruby-version` is more like convenience, I don't think it should be the source of what `RUBY_VERSION` TruffleRuby reports.

Reply:
Ok I'm not massively fussed about `LANGUAGE_VERSION` so I'll remove that.

Reply:
Done.